src/test/java/org/jclouds/vcloud/director/v1_5/AbstractVAppApiLiveTest.java

```diff
@@ -136,13 +138,13 @@ protected void setupEnvironment() {
       assertNotNull(vAppTemplate, String.format(ENTITY_NON_NULL, VAPP_TEMPLATE));
 
       // Instantiate a new VApp
-      VApp vAppInstantiated = instantiateVApp();
+      VApp vAppInstantiated = composeVApp();
```

Comment:
@andreaturli I replaced instantiateVapp with composeVapp because `composeVapp` is the call used in `VCloudDirectorComputeServiceAdapter#createNodeWithGroupEncodedIntoName`